pkg/storage/utils/volumes.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package utils
+
+import (
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "kubevirt.io/api/core/v1"
+	snapshotv1 "kubevirt.io/api/snapshot/v1beta1"
+
+	backendstorage "kubevirt.io/kubevirt/pkg/storage/backend-storage"
+)
+
+const (
+	FLAG_INCLUDE_ALL = 0
+	FLAG_EXCLUDE_EFI = 1 << iota
+	FLAG_EXCLUDE_TPM
```

Comment:
I don't understand the need for an EFI/TPM distinction here. They're both stored in the same PVC, and you may or may not want to include that PVC.
If, for example, a VM has both persistent EFI and persistent TPM and the snapshot code sets `FLAG_EXCLUDE_EFI` and **not** `FLAG_EXCLUDE_TPM`, we're still snapshotting both, since they're in the same PVC.